### PR TITLE
Vendor dependencies and more

### PR DESCRIPTION
**Pull request by danmrichards**
## Changes
* Vendored dependencies.
* Fixed typo referencing env.yaml
* Updated `KeyValuePairs` string method. Displayed confusing default value on -h.
* Fixed definition of headers in examples, tests and README.
* Added explicit support for query parameters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,14 +80,15 @@ name= "httpbin post - returns post data"
 method= "POST"
 url="http://{{.base_service_url}}/post"
 wants_code= 200
-headers=["Content-Type: application/json"]
```
Comment (danmrichards) [inline README.md:80]: You'll see this change elsewhere too. But this toml does not correct unmarshal to `map[string]string` as expected.

A table like this does though
```toml
[litmus.test.headers]
Content-Type = "application/json"
```

File: README.md [hunk 2]
```diff
@@ -60,6 +60,9 @@ The `*_test.toml` files contain the requests that will be made.  They're execute
 	method="GET"
 	url="http://{{.base_service_url}}/get"
 	want_code=200
+[litmus.test.query]
```
Comment (danmrichards) [inline README.md:63]: Of course we could just add the query params to the `url` and it'd work pretty well. But I feel that explicitly defining them like this is more readable and testable

File: pkg/keyValuePairs.go [hunk 3]
```diff
@@ -16,7 +16,7 @@ type KeyValuePairs []KeyValuePair
 
 // String returns the string representation of KeyValuePairs.
 func (k *KeyValuePairs) String() string {
-	return fmt.Sprintf("%#+v", k)
+	return ""
```
Comment (danmrichards) [inline pkg/keyValuePairs.go:19]: This is a contentious one. The current implementation shows `(default &pkg.KeyValuePairs(nil))` as the default value when running with the `-h` flag.

I don't feel thats a very helpful default, especially for a non-developer. The `String` method seems to only be here to satisfy an interface so imo a zero value string is just as useful.

File: README.md (hunk 2 above)
Comment (codingconcepts) [reply README.md:63]: And we can still use query params for the die-hards, so yep, anything that can make the tool easier to use the better it’ll be.

**Review by codingconcepts — APPROVED** [review]

**Pull request merged**